faqs.md

```diff
@@ -21,13 +21,20 @@ If there’s already an [existing proposal for grants](https://github.com/aragon
 - Make sure the [Proposals for grants](https://github.com/aragon/nest/issues) doesn’t have an existing proposals for something similar to your proposal. If there is an existing one that is close to your idea, join in the discussion of that Proposal to see if yours could be integrated into the existing one, or if you should create a new proposal
 - If nothing similar exists, create [a new Proposal for grants](https://github.com/aragon/nest/issues/new) at the [Nest Repository](https://github.com/aragon/nest/) and fill it out in detail following the [Guide for submitting a proposal for grants](guides/guide_for_submitting_a_proposal_for_grants.md)
 
-## **Who will be deciding on how the grants are distributed?**
-The funding will be described in the Proposal of the grant. The total amount, how many portions will it be divided into, timetable and milestones will be up for discussion.
+## **How does the application process works?**
+The application process has two stages:
+
+1. The proposal stage (see above). The main focus in this stage is the discussion about the idea/project and whether it should be considered for a grant. Once the proposal is approved we move to the second stage. 
```

Comment:
"whether or not it" could be a better wording

Reply:
Also lines 9, 17 and 22 have broken links to guides, they have been moved so the links should be fixed.

Reply:
And L12 has broken links to `team.md` & `roadmap.md` templates.

Reply:
can you fix that?